src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\TestCase;
```

Comment:
`<4.8.35|<5.4.3,>=5.0`  is in the conflict section of composer.json

What it really means is that we want PHPUnit 4.8.35 (the last version of PHPUnit 4) or a PHPUnit>=5.4.3
As you noticed, 5.4.3 is the first version of PHPUnit5 with the namespace ` PHPUnit\Framework\TestCase`